Formula/filebeat.rb

```diff
@@ -25,12 +17,15 @@ def install
     cd gopath/"src/github.com/elastic/beats/filebeat" do
       system "make"
       libexec.install "filebeat"
-      etc.install "etc/filebeat.yml"
+
+      (etc/"filebeat").install "filebeat.yml"
+      (etc/"filebeat").install "filebeat.template.json"
+      (etc/"filebeat").install "filebeat.template-es2x.json"
     end
 
     (bin/"filebeat").write <<-EOS.undent
       #!/bin/sh
-      exec "#{libexec}/filebeat" -c "#{etc}/filebeat.yml" "$@"
+      exec "#{libexec}/filebeat" --path.config "#{etc}/filebeat" --path.home="#{prefix}" --path.logs="#{var}/log/filebeat" --path.data="#{opt_prefix}" "$@"
```

Comment:
The files in /usr/local/opt are symlinks that change every time the installation is upgraded to point to the current Cellar path. So the path is stable, but the directory it points to is changing every time.

Anything that persists across versions probably belongs in one of
```
etc
etc/"foo"
var
etc/"foo"
```
etc and var are different from bin, lib, include, share, ... in that they are locations in the `HOMEBREW_PREFIX` not the formula `prefix`. So `etc` is the directory `/usr/local/etc` for all formulae and `var` is the directory `/usr/local/var` for all formulae.